### PR TITLE
Adding a PivotTable sample

### PR DESCRIPTION
**Pull request by AlexJerabek**
There was a customer issue question around getting data from a PivotTable. This sample should help illustrate how to do that scenario.

Link to staging site: https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/excel-samples?branch=AlexJ-Pivot#log-the-grand-total-values-from-a-pivottable

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[856dac5](https://github.com/OfficeDev/office-scripts-docs/commits/856dac5d71d67bafd821c810c4fb3b2b4fd167a7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/sample-pivottable-grand-total-row.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Pivot/docs/images/sample-pivottable-grand-total-row.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Pivot/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C17%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202004171828020852-104%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=YYkDpfmYbOXyb7uFKZocZpDYjSzY1aFWDk3Zl%2BB9T9M%3D&st=2020-04-17T18%3A24%3A14Z&se=2020-05-18T18%3A29%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a830b45](https://github.com/OfficeDev/office-scripts-docs/commits/a830b45033c052cda2d89064a9c792d54a5d6a34)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/sample-pivottable-grand-total-row.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Pivot/docs/images/sample-pivottable-grand-total-row.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Pivot/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C17%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202004171828251581-104%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=N2nKXaIRD5kDkPdSZhmpsbJdOgVgN5HdA9p7iELiBXw%3D&st=2020-04-17T18%3A25%3A41Z&se=2020-05-18T18%3A30%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/resources/excel-samples.md [hunk 1]
```diff
@@ -123,6 +123,40 @@ async function main(context: Excel.RequestContext) {
...
+  let pivotDataRange = pivotTable.layout.getDataBodyRange();
+
+  // Get the range with the "grand totals" for the PivotTable columns.
+  let grandTotalRange = pivotDataRange.getLastRow();
+  grandTotalRange.load("values");
+  await context.sync();
+  
+  // Print each of the "Grand Totals" to the console.
+  grandTotalRange.values[0].forEach((column, columnIndex) => {
+    console.log(`Grand total of ${pivotColumnLabelRange.values[0][columnIndex]}: ${grandTotalRange.values[0][columnIndex]}`);
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/excel-samples.md:155]: Can you include as a comment below what the output would be in this case? I feel like that would be useful.

Comment (ElizabethSamuel-MSFT) [reply docs/resources/excel-samples.md:155]: Maybe just the first instance as an example

**Comment by o365devx** [comment]
Docs Build status updates of commit _[db66882](https://github.com/OfficeDev/office-scripts-docs/commits/db6688235f736588eed43c264bff3f75151d2f08)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/sample-pivottable-grand-total-row.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Pivot/docs/images/sample-pivottable-grand-total-row.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Pivot/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C17%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202004172112066072-104%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=0Gp7sbtFYOeDQbBuza5bYAQTFg%2FaGf71tY32V9%2BD6Y4%3D&st=2020-04-17T21%3A08%3A18Z&se=2020-05-18T21%3A13%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/resources/excel-samples.md (hunk 1 above)
Comment (AlexJerabek) [reply docs/resources/excel-samples.md:155]: Good call! Updated.

**Pull request merged**